### PR TITLE
Adds head tag CRUD support.

### PR DESCRIPTION
**Pull request by simonprickett**
Adds support for:

* `GET /web/headtags`
* `GET /web/headtags/HEADTAG_ZUID`
* `POST /web/headtags`
* `PUT /web/headtags/HEADTAG_ZUID`
* `DELETE /web/headtags/HEADTAG_ZUID`

### CODE REVIEW

File: zestyio-api-wrapper.js [hunk 1]
```diff
@@ -475,6 +480,45 @@ class ZestyioAPIWrapper {
     return await this.getRequest(settingURL)
   }
 
+  async createHeadTag(tag) {
```
Comment (ardeay) [inline zestyio-api-wrapper.js:486]: Please add an example of this post json body to the docs or link to postman docs in readme.

**Review by ardeay — CHANGES_REQUESTED** [review]
Please add examples and/or a link to the postman docs in the readme.

File: zestyio-api-wrapper.js (hunk 1 above)
Comment (simonprickett) [reply zestyio-api-wrapper.js:486]: Done in https://github.com/zesty-io/zestyio-node-api-wrapper/pull/11/commits/0f2b019a0154c8a71f9b88c495dc76b016161e86

**Comment by ardeay** [comment]
thank you @simonprickett merged

**Pull request merged**